pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -464,3 +466,34 @@ func (m *cgroupManagerImpl) ReduceCPULimits(cgroupName CgroupName) error {
 	}
 	return m.Update(containerConfig)
 }
+
+func getStatsSupportedSubsytems(cgroupPaths map[string]string) (*libcontainercgroups.Stats, error) {
```

Comment:
ideally, in the future we can move this or delegate to what ever we do for on-demand usage stat collection.